ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Node/NodePageVulnerabilities.tsx

```diff
@@ -48,6 +59,30 @@ function NodePageVulnerabilities({ nodeId }: NodePageVulnerabilitiesProps) {
                 <Text>Review and triage vulnerability data scanned on this node</Text>
             </PageSection>
             <PageSection isFilled className="pf-v5-u-display-flex pf-v5-u-flex-direction-column">
+                <SummaryCardLayout isLoading={summaryRequest.loading} error={summaryRequest.error}>
+                    <SummaryCard
+                        loadingText={'Loading node CVEs by severity summary'}
+                        data={summaryRequest.data}
+                        renderer={({ data }) => (
+                            <BySeveritySummaryCard
+                                title="CVEs by severity"
+                                severityCounts={data.node.cveCountBySeverityAndFixability}
+                                hiddenSeverities={hiddenSeverities}
+                            />
+                        )}
+                    />
+                    <SummaryCard
+                        loadingText={'Loading node CVEs by status summary'}
+                        data={summaryRequest.data}
+                        renderer={({ data }) => (
+                            <CvesByStatusSummaryCard
+                                cveStatusCounts={data.node.cveCountBySeverityAndFixability}
+                                hiddenStatuses={hiddenStatuses}
+                                isBusy={summaryRequest.loading}
```

Comment:
Observing, not necessarily requesting any change: `isBusy` prop here but not in preceding sibling card (which was written several months earlier).

I read that as never stop improving, as home improvement store encourages?

Reply:
I noticed this too, and believe this is a side effect of some older patterns in the query loading and test code. I've got it on my list to revisit as I'm not 100% sure it is still needed.

Reply:
I'm wondering whether we should name this property `isLoading` instead of `isBusy`. Although internally it aligns with the `aria-busy` attribute, does a user of this component need to understand its internal use, or should they simply provide what's required? This distinction might enhance clarity for those implementing the component. Just an initial thought I had looking at the prop

Reply:
Yeah I agree, `isLoading` would be consistent with naming for similar properties elsewhere. When I check to see if this is deprecated by the new loading patterns I'll either remove it or rename it depending on what I find.